pkg/resourcemanager/azuresql/azuresqluser/azuresqluser.go

```diff
@@ -204,15 +204,17 @@ func (s *AzureSqlUserManager) DeleteSecrets(ctx context.Context, instance *v1alp
 
 		for _, formatName := range customFormatNames {
 			key := types.NamespacedName{Namespace: secretKey.Namespace, Name: instance.Name + "-" + formatName}
-
 			err = secretClient.Delete(
 				ctx,
 				key,
 			)
 			if err != nil {
-				instance.Status.Message = "failed to delete secret, err: " + err.Error()
-				return false, err
+				if !strings.Contains(err.Error(), "does not exist") {
+					instance.Status.Message = "failed to delete secret, err: " + err.Error()
```

Comment:
Since this is a user facing error message, perhaps
```suggestion
					instance.Status.Message = "Failed to delete secret: " + err.Error()
```

Reply:
Yeah, I think the majority of errors we set in Status that don't come directly from Azure are likely not capitalized...not necessarily opposed but changing this one probably won't do much more than add to the inconsistency.